app/components/DeleteButton/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+*
+* DeleteButton
+*
+*/
+
+import React from 'react';
+
+import styles from './styles.css';
+
+function DeleteButton(props) {
+  return (
+    <button
+      className={styles.deleteButton}
+      onClick={() => {
+        const response = window.confirm(props.confirmText); // eslint-disable-line no-alert
+        if (response) {
+          props.onClick();
```

Comment:
Goodie, that's necessary! 

I suggest implementing some alert/notiification component across the app instead. See the above comment regarding the ESlinting for an explanation.

It can be done by creating some library yourself, by building on modals from [some UX framework](https://semantic-ui.com/modules/modal.html) that you use (don't remember what you use here now). Then you can, across your app, do i.e. `userNotification.confirm('Delete?', yesMethod, noMethod)`, and `userNotification.message('Your shit is deleted')`.

Or, if you don't want to reinvent the wheel (which you don't, I assume), there are [plenty](https://github.com/igorprado/react-notification-system) of [libraries](https://github.com/minhtranite/react-notifications) for these things. =) Look around, and find something that suits this project, and is maintained (to some extent). If you don't, I think you'll have to reinvent the wheel ;)